src/style/product-card.css

```diff
@@ -0,0 +1,119 @@
+.product-card {
+  --color-acdc: #00acdc;
+  --color-white: #fff;
+  --color-black: #060b35;
+  --color-gray: #616070;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  font-family: Roboto, Arial, sans-serif;
+  background-color: var (--coloor-white);
+  padding: 32px 16px 16px;
+}
+
+.product-card__img {
+  text-align: center;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.product-card__description-model {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: var(--color-black);
+}
+
+.product-card__description-id {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var( --color-gray);
+  margin-bottom: 16px;
+}
+
+.product-card__comment {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+  width: 100%;
+
+}
+
+.product-card__comment-amount {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-black);
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+  width: 100%;
+}
+
+.product-card__price-item {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-gray);
+}
+
+.product-card__price-amount {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color-black);
+}
+
+.product-card__button {
+  text-align: center;
+  line-height: 38px;
+  width: 100%;
+
+}
+
+.product-card__button-link {
+  display: block;
+  box-sizing: border-box;
+  width: 100%;
+  background-color: var(--color-acdc);
+  color: var(--color-white);
+  text-decoration: none;
+  border: 1px solid  var(--color-acdc);
+  border-radius: 5px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  text-align: center;
+  line-height: 38px;
+
+}
+
+.product-card__button-link:hover {
+  display: block;
+  width: 100%;
+  background-color: var(--color-white);
+  color: var(--color-acdc);
+  border: 1px solid  var(--color-acdc);
+  border-radius: 5px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
```

Comment:
You should change only 2 properties on hover state: background-color and text color, delete other styles, that don't change